doc/_static/diagrams/git_setup.dot

```diff
@@ -0,0 +1,64 @@
+digraph gitstructure {
+    /* LAYOUT CONTROL */
+    rankdir=UD
+    newrank=true
+    ranksep=0.75
+    clusterrank=local
+    labelloc=t
+    pad=0.1
+
+    /* GLOBAL TITLE */
+    label="Schematic of recommended setup"
+
+    /* GLOBAL STYLES */
+    splines=true
+    fontsize=20
+    fontname="Source Sans Pro"
+
+    node [shape=box
+          style=filled
+          fontsize=12
+          fontname="Source Sans Pro"
+          fillcolor="#bbbbbb"
+          margin="0.35,0.1"]
```

Comment:
Do you need/want links anywhere? If so you can get them in Graphviz, for example for nodes or edges you can do (in Python syntax here):
```
            edge.attr['URL'] = '../overview.html'
            edge.attr['target'] = '_top'
```
You need to render in SVG mode. The `..` in the link is necessary to get out of the SVG itself.

Reply:
I'm not sure there's a lot of value in creating links from this diagram. The `git clone` and `git remote add` commands are shown in the very next section, and there's no way to link to an unknown user's fork or to their local computer, so the only link that makes sense is the first node (the main MNE-Python repo).